.circleci/config.yml

```diff
@@ -18,25 +18,73 @@ aliases:
         command: ./scripts/cibuild
     - save_cache: *save_sbt_cache
 
+  - &run_cipublish
+    - checkout
+    - run: echo "${SCALA_VERSION}" > /tmp/scala_version
+    - restore_cache: *restore_sbt_cache
+    - run:
+        name: "Import signing key"
+        command: |
+          echo "${GPG_KEY}" | base64 -d > signing_key.asc && \
+          gpg --batch \
+            --passphrase "${GPG_PASSPHRASE}" \
+            --import signing_key.asc
+    - run:
+        name: Executing cipublish
+        command: ./scripts/cipublish
+
+  # Build environments
+  - &openjdk8-scala2_11_12-nodelts_environment
+    docker:
+      - image: circleci/openjdk:8-stretch-node
+    environment:
+      SCALA_VERSION: 2.11.12
+
+  - &openjdk8-scala2_12_8-nodelts_environment
+    docker:
+      - image: circleci/openjdk:8-stretch-node
+    environment:
+      SCALA_VERSION: 2.12.8
+
 version: 2
 workflows:
   version: 2
   build:
     jobs:
       - "openjdk8-scala2.11.12-nodelts"
       - "openjdk8-scala2.12.8-nodelts"
+      - "openjdk8-scala2.11.12-nodelts_deploy":
+          requires:
+            - "openjdk8-scala2.11.12-nodelts"
+          filters:
+            branches:
+              only:
+                - develop
+                - master
```

Comment:
I believe we still need to target `master`. For example, adding

```yaml
tags:
  only:
    - /^(.*)$/
```

but setting branches to:

```yaml
branches:
  only:
    - develop
```

will only build commits on `develop` as well as tagged commits on `develop`.

It's good you brought this up, because I don't think tagged commits would ever have actually been built. It looks like builds for tagged commits are disabled by default, and you need to apply a filter in order to enable them.

See: https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag